### ISSUE TITLE
ssl_verify: false seems to be ignored

### ISSUE THREAD

**Issue by mparry** [conda/conda#6872]
The `ssl_verify` option in conda doesn't seem to work in conda 4.4.8+. (I haven't tested preceding versions, just 4.4.8 and 4.4.10, but it certainly used to work some time ago.)

I have the option enabled currently:

```none
> conda config --get ssl_verify

--set ssl_verify False
```

And yet:

```none
> conda update foo

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.
SSLError(MaxRetryError('HTTPSConnectionPool(host=\'an-internal-server\', port=443):
Max retries exceeded with url: /repository/some-repo/win-64/repodata.json
(Caused by SSLError(SSLError("bad handshake: Error([(\'SSL routines\', \'ssl3_get_server_certificate\', \'certificate verify failed\')],)",),))',),)
```

If I access this same URL using `requests` directly instead then it's fine:

```python
url = "https://an-internal-server/repository/some-repo/win-64/repodata.json"
requests.get(url, verify=False)
<Response [200]>
```

This may be Windows-specific, as it seems fine on my Mac with identical configuration. Here's the user agent string reported by `conda info`:

```
user-agent : conda/4.4.10 requests/2.18.4 CPython/3.6.4 Windows/7 Windows/6.1.7601
```

**Comment by kalefranz** [comment]
Out of curiosity, does the `--insecure` command line flag work for you?

**Comment by mparry** [comment]
No, `--insecure` made no difference.

However, typically, looking at it again after finally reporting a ticket, I've now figured out what the problem was. Many months ago I set the `REQUESTS_CA_BUNDLE` environment variable (for a different internal setup that no longer applies) and that seems to take precedence. Unsetting that gets it working again. 

I guess conda could warn about this situation, if you really wanted, but the original bug report is invalid and I'll close the ticket.
